database/migrations/sqls/20210219043110-tree-up.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE tree
+(
+    id uuid NOT NULL PRIMARY KEY,
+    latest_capture_id uuid NOT NULL,
+    image_url varchar NOT NULL,

```

Comment:
`image_url` here is copy of the `image_url` stored in `latest_capture_id`. We could omit this column from here. But for convenience sake we could leave it here.